### PR TITLE
fix(bivariate_matrix): 17663 no legend for bivariate matrix layers

### PR DESCRIPTION
**Pull request by vkozio**
fix bivariate layer management
bivariate matrix migrated to reatom 3

## Summary by CodeRabbit

- **New Features**
	- Introduced `ast-grep` patterns for easier code migration between Reatom versions.
	- Added dynamic import for Sentry initialization in `App.tsx` for improved error tracking.
	- New atoms and actions for managing bivariate matrix selections and numerators, enhancing state management.
	- Launched a new feature for bivariate data management, including a set of related atoms for logical layers and resources.

- **Refactor**
	- Updated condition for Sentry initialization based on the environment in `App.tsx`.
	- Added a check for map source existence before setting feature states to prevent errors.
	- Adjusted `configureStore` to improve logger subscription based on development environment.
	- Refactored bivariate manager components and atoms to use Reatom for state management, improving code efficiency and readability.

- **Bug Fixes**
	- Fixed potential issue with uninitialized map sources in feature state setting.

- **Documentation**
	- Added comprehensive summaries for each change to facilitate understanding of updates and refactoring efforts.

### CODE REVIEW

File:  LICENSE
Comment (albaranau) [inline  LICENSE:1]: Was it planned to remove the license file? Don't we need a different one to replace it?

File: src/core/logical_layers/renderers/BivariateRenderer/activeAndHoverFeatureStates.ts [hunk 1]
```diff
@@ -32,6 +32,11 @@ export function createFeatureStateHandlers(ctx: {
     featureId: string | number,
     newState: Partial<Record<keyof typeof FEATURE_STATES, boolean>>,
   ) => {
+    const s = ctx.map.getSource(ctx.sourceId);
```
Comment (albaranau) [inline src/core/logical_layers/renderers/BivariateRenderer/activeAndHoverFeatureStates.ts]: Maybe it's better to give s a more meaningful name?

File:  LICENSE
Comment (Komzpa) [reply  LICENSE:1]: it looks like we had "LICENSE" and " LICENSE". The correct text is the one that has "Kontur Inc" in it and it has to be in the file without space in front.

Comment (albaranau) [reply  LICENSE:1]: @vkozio seems like the wrong License file was deleted?

**Review by albaranau — APPROVED** [review]

**Pull request merged**